dev/Dockerfile.dev

```diff
@@ -19,7 +19,7 @@ COPY Makefile Makefile
 COPY setup.cfg .
 COPY dev/entrypoint.sh ./
 
-COPY migrations/migrate.py migrations/.
+COPY migrations/*.py migrations/
 COPY migrations/migrations migrations/migrations/.
```

Comment:
Wouldn't it be better to mount these folders?

Reply:
I thought about it, but when *developing* migrations, there is a dedicated setup in the migrations folder. Until you want to change the intersection (load_modules), you have to rebuild/test-edit with vim. I had to rebuild 3 times, which is a low quote I guess to make it work. I think there is currently no need

Reply:
I found that mounting is almost always better than copying because it eliminates one source of programming mistakes and just makes the experience a lot smoother if you need it. But I see your point

Reply:
I mounted it now ;)